lib/formatter.js

```diff
@@ -136,20 +136,24 @@ function formatOrderedList(elem, fn, options) {
     a: function(start, i) { return String.fromCharCode(i + start + 97)},
     A: function(start, i) { return String.fromCharCode(i + start + 65)}
   };
-  // Determine type
-  var olType = elem.attribs.type || '1'
+  // TODO Imeplement the other valid types
+  //   Fallback to type '1' function for other valid types
+  typeFunctions.i = typeFunctions['1'];
+  typeFunctions.I = typeFunctions['1'];
```

Comment:
I think it is the best to simply fall back on any other type instead of only these two cases. This would make `html-to-text` more robust against invalid types.